### DISCUSSION TITLE
[Feature]: Swipe down to refresh in mobile app

### DISCUSSION THREAD

**Discussion by JVT038** [immich-app/immich#1662] [Feature Request]
### Feature detail

Add a feature that allows the user to manually refresh the app by swiping down, which shows a spinning wheel and then refreshes the timeline.

### Platform

Mobile App

**Comment by jrasm91** [comment]
Implemented in #2085 and released in [v1.52](https://github.com/immich-app/immich/releases/v1.52.0).
